modules/telemetry/src/specific/planetssonification.cpp

```diff
@@ -0,0 +1,779 @@
+    _saturnProperty.ringsEnabled.onChange([this]() { onSaturnSettingChanged(); });
+    addPropertySubOwner(_saturnProperty);
+
+    // Uranus
+    _uranusProperty.toggleAll.onChange([this]() { onUranusAllChanged(); });
+    _uranusProperty.sizeDayEnabled.onChange([this]() { onUranusSettingChanged(); });
+    _uranusProperty.gravityEnabled.onChange([this]() { onUranusSettingChanged(); });
+    _uranusProperty.temperatureEnabled.onChange([this]() { onUranusSettingChanged(); });
+    _uranusProperty.atmosphereEnabled.onChange([this]() { onUranusSettingChanged(); });
+    _uranusProperty.moonsEnabled.onChange([this]() { onUranusSettingChanged(); });
+    // No rings visible in OpenSpace
+    addPropertySubOwner(_uranusProperty);
+
+    // Neptune
+    _neptuneProperty.toggleAll.onChange([this]() { onNeptuneAllChanged(); });
+    _neptuneProperty.sizeDayEnabled.onChange([this]() { onNeptuneSettingChanged(); });
+    _neptuneProperty.gravityEnabled.onChange([this]() { onNeptuneSettingChanged(); });
+    _neptuneProperty.temperatureEnabled.onChange([this]() { onNeptuneSettingChanged(); });
+    _neptuneProperty.atmosphereEnabled.onChange([this]() { onNeptuneSettingChanged(); });
+    _neptuneProperty.moonsEnabled.onChange([this]() { onNeptuneSettingChanged(); });
+    // No rings visible in OpenSpace
+    addPropertySubOwner(_neptuneProperty);
+
+    // Precision
+    _anglePrecision = _precisionProperty.lowAnglePrecision;
+    _distancePrecision = _precisionProperty.lowDistancePrecision;
+    addPropertySubOwner(_precisionProperty);
+}
+
+PlanetsSonification::~PlanetsSonification() {
+    stop();
+}
+
+PlanetsSonification::PlanetProperty::PlanetProperty(
+                                  properties::PropertyOwner::PropertyOwnerInfo planetInfo)
+    : properties::PropertyOwner(planetInfo)
+    , toggleAll(PlanetToggleAllInfo, false)
+    , sizeDayEnabled(SizeDayInfo, false)
+    , gravityEnabled(GravityInfo, false)
+    , temperatureEnabled(TemperatureInfo, false)
+    , atmosphereEnabled(AtmosphereInfo, false)
+    , moonsEnabled(MoonsInfo, false)
+    , ringsEnabled(RingsInfo, false)
+{
+    // Add the common settings
+    addProperty(toggleAll);
+    addProperty(sizeDayEnabled);
+    addProperty(gravityEnabled);
+    addProperty(temperatureEnabled);
+
+    // Add special settings for planets that support it
+    // Mercury is the only planet that does not have any form of atmosphere
+    if (planetInfo.guiName != "Mercury") {
+        addProperty(atmosphereEnabled);
+    }
+    // Mercury and Venus are the only planets that does not have any moon
+    if (planetInfo.guiName != "Mercury" && planetInfo.guiName != "Venus") {
+        addProperty(moonsEnabled);
+    }
+    // Saturn is the only planet with rings (visible in OpenSpace)
+    if (planetInfo.guiName == "Saturn") {
+        addProperty(ringsEnabled);
+    }
+}
+
+PlanetsSonification::PrecisionProperty::PrecisionProperty(
```

Comment:
```suggestion
PlanetsSonification::PrecisionProperties::PrecisionProperties(
```
See previous comment